new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItElkSample.java

```diff
@@ -0,0 +1,122 @@
+@IntegrationTest
+class ItElkSample {
+
+  // constants for Operator
+  private static String opNamespace = null;
+  private static String domainNamespace = null;
+  private static ConditionFactory withStandardRetryPolicy = null;
+
+  // constants for ELK Stack
+  private static LoggingExporterParams elasticsearchParams = null;
+  private static LoggingExporterParams kibanaParams = null;
+  private static LoggingFacade logger = null;
+
+  /**
+   * Install Elasticsearch, Kibana and Operator, create a one cluster domain.
+   *
+   * @param namespaces list of namespaces created by the IntegrationTestWatcher by the
+   *                   JUnit engine parameter resolution mechanism
+   */
+  @BeforeAll
+  public static void init(@Namespaces(2) List<String> namespaces) {
+    logger = getLogger();
+    // create standard, reusable retry/backoff policy
+    withStandardRetryPolicy = with().pollDelay(2, SECONDS)
+      .and().with().pollInterval(10, SECONDS)
+      .atMost(5, MINUTES).await();
+
+    // get a new unique opNamespace
+    logger.info("Assigning a unique namespace for Operator");
+    assertNotNull(namespaces.get(0), "Namespace list is null");
+    opNamespace = namespaces.get(0);
+
+    // get a new unique domainNamespace
+    logger.info("Assigning a unique namespace for Domain");
+    assertNotNull(namespaces.get(1), "Namespace list is null");
+    domainNamespace = namespaces.get(1);
+
+    // install and verify Elasticsearch
+    logger.info("install and verify Elasticsearch");
+    elasticsearchParams = assertDoesNotThrow(() -> installAndVerifyElasticsearch(),
+            String.format("Failed to install Elasticsearch"));
+    assertTrue(elasticsearchParams != null, "Failed to install Elasticsearch");
+
+    // install and verify Kibana
+    logger.info("install and verify Kibana");
+    kibanaParams = assertDoesNotThrow(() -> installAndVerifyKibana(),
+        String.format("Failed to install Kibana"));
+    assertTrue(kibanaParams != null, "Failed to install Kibana");
+
+    // install and verify Operator
+    installAndVerifyOperator(opNamespace, opNamespace + "-sa", false, 0, true, domainNamespace);
+  }
+
+  /**
+   * Uninstall ELK Stack and delete domain custom resource.
+   */
+  @AfterAll
+  void tearDown() {
+    // uninstall ELK Stack
+    if (elasticsearchParams != null) {
+      logger.info("Uninstall Elasticsearch pod");
+      assertDoesNotThrow(() -> uninstallAndVerifyElasticsearch(elasticsearchParams),
+          "uninstallAndVerifyElasticsearch failed with ApiException");
+    }
+
+    if (kibanaParams != null) {
+      logger.info("Uninstall Elasticsearch pod");
+      assertDoesNotThrow(() -> uninstallAndVerifyKibana(kibanaParams),
+          "uninstallAndVerifyKibana failed with ApiException");
+    }
+  }
+
+  /**
+   * Verify that ELK Stack is ready to use by checking the index status of
+   * Kibana and Logstash creaed in the Operator pod successfully.
```

Comment:
typo creaed -> created
Can u provide more detail step.
  a. Start the operator with Elk Enabled
  b. Install Kibana
  c. Start the WebLogic Domain
  d. ....

Reply:
This is a good idea. how about to add this detailed steps to junit5-elk-tests, https://github.com/oracle/weblogic-kubernetes-operator/pull/1848, because I may remove this class after this PR is merged

Reply:
I just added the comments below to ItElasticLogging in junit5-elk-tests, PR https://github.com/oracle/weblogic-kubernetes-operator/pull/1848

/**
 * To test ELK Stack used in Operator env, this Elasticsearch test does
 * 1. Install Kibana/Elasticsearch 
 * 2. Install and Start the operator with Elk Enabled
 * 3. Verify that ELK Stack is ready to use by checking the index status of
 *    Kibana and Logstash created in the Operator pod successfully
 * 4. Create and Start the WebLogic domain
 * 5. Verify that Elasticsearch collects data from WebLogic logs and
 *    stores them in its repository correctly.
 */

Reply:
Ok.